onboarding/mentors.md

```diff
@@ -3,4 +3,53 @@ title: Onboarding Responsibilities for Mentors of New Hires
 description: How to be an effective mentor of a new hire
 ---
 
-TBD
+# Mentorship at Artsy
+
+### Historical Context
+- TODO: talk to Orta/Sarah/Joey about how we used to do it.
+- [Alan's post](http://artsy.github.io/blog/2016/09/22/professional-development-at-artsy-engineering/)
```

Comment:
#minor The link should describe what the article is about. So something like "Blog post on professional development at Artsy"